formatter/formatter.go

```diff
@@ -105,6 +104,9 @@ func FormatStepWithResolvedArgs(step *gauge.Step) string {
 				formattedArg := fmt.Sprintf("\"%s\"", stepFragmet.GetParameter().Value)
 				text = strings.Replace(text, gauge.ParameterPlaceholder, formattedArg, 1)
 			}
+			if argument.ArgType == gauge.Static || argument.ArgType == gauge.TableArg{
+				return FormatStep(step)	
```

Comment:
We are trying to return a formatted step if one of the parameters is static or table argument. What will be the case when you have a mixture of static arguments and dynamic arguments? Is this case handled?